gradlew build check test failing

I followed the instructions to install and build synthea and it failed. There were multiple occurrences of this error message:

java.lang.IllegalArgumentException: resource generic/lookup_tables/covid19_prob.csv not found.

That file exists, but not in that location. 

Once I learn more about the project, I will attempt to fix the configuration problem for the tests and create a pull request, but right now all I can do is report the error.

thank you

George Lilly

I am using the developer instructions.
git clone https://github.com/synthetichealth/synthea.git
cd synthea
./gradlew build check test

<snip>
7: warning: no @return
  public Double getVitalSign(VitalSign vitalSign, long time) {
                ^
/home/glilly/synthea/src/main/java/org/mitre/synthea/world/agents/Person.java:467: warning: no @param for vitalSign
  public void setVitalSign(VitalSign vitalSign, double value) {
              ^
100 warnings

> Task :test

org.mitre.synthea.engine.GeneratorTest > testGeneratePeopleDefaultLocation FAILED
    java.lang.RuntimeException: java.lang.IllegalArgumentException: resource generic/lookup_tables/covid-19-severity-outcomes.csv not found.
        at org.mitre.synthea.engine.Module$ModuleSupplier.get(Module.java:424)
        at org.mitre.synthea.engine.Module.lambda$getModules$4(Module.java:233)
        at java.util.concurrent.ConcurrentHashMap.forEach(ConcurrentHashMap.java:1597)
        at org.mitre.synthea.engine.Module.getModules(Module.java:231)
        at org.mitre.synthea.engine.Generator.init(Generator.java:233)
        at org.mitre.synthea.engine.Generator.<init>(Generator.java:167)
        at org.mitre.synthea.engine.Generator.<init>(Generator.java:117)

<snip>


org.mitre.synthea.engine.GeneratorTest > testUpdateAfterCreationAndSerialization
 FAILED
    java.lang.RuntimeException: java.lang.IllegalArgumentException: resource generic/lookup_tables/covid-19-severity-outcomes.csv not found.
        at org.mitre.synthea.engine.Module$ModuleSupplier.get(Module.java:424)
        at org.mitre.synthea.engine.Module.lambda$getModules$4(Module.java:233)
        at java.util.concurrent.ConcurrentHashMap.forEach(ConcurrentHashMap.java:1597)  

<snip>

37 tests completed, 9 failed, 2 skipped

> Task :test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> Process 'Gradle Test Executor 1' finished with non-zero exit value 137
  This problem might be caused by incorrect test process configuration.
  Please refer to the test execution section in the User Manual at https://docs.gradle.org/6.1.1/userguide/java_testing.html#sec:test_execution

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.1.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 9m 2s
15 actionable tasks: 8 executed, 7 up-to-date
[glilly@39505519c969 synthea]$

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 org.gradle.jvmargs=-Xmx2048m

